src/index.ts

```diff
@@ -107,16 +108,23 @@ export default class ArchiveServer {
     if (content && status >= 200 && status < 300) {
       let { text, encoding, mimeType } = content;
       let body: Buffer | undefined;
+
+      let isText = mimeType.match(/text|javascript/) !== null;
```

Comment:
Where is this used?

Reply:
I think it was leftover from the previous implementation, before we switched to using `mimeTypes`